eq_cir_proxy_service/services/instrument/instrument_retrieval_service.py

```diff
@@ -54,7 +54,7 @@ async def retrieve_instrument(instrument_id: UUID) -> Instrument:
         async with AsyncClient() as client:
             response = await client.get(url, params={"guid": str(instrument_id)})
     except RequestError as e:
-        logger.exception("Error occurred while retrieving instrument: %s")
+        logger.exception("Error occurred while retrieving instrument: ", error=e)
```

Comment:
```suggestion
        logger.exception("Error occurred while retrieving instrument.", error=e)
```
Removes `:`, as this no longer uses `%s`.